torch/onnx/_internal/onnx_proto_utils.py

```diff
@@ -163,47 +164,12 @@ def export_data(data, value_info_proto, f: str) -> None:
 def _export_file(
     model_bytes: bytes,
     f: io.BytesIO | str,
-    export_type: str,
     export_map: Mapping[str, bytes],
 ) -> None:
     """export/write model bytes into directory/protobuf/zip"""
-    if export_type == _exporter_states.ExportTypes.PROTOBUF_FILE:
-        assert len(export_map) == 0
-        with torch.serialization._open_file_like(f, "wb") as opened_file:
-            opened_file.write(model_bytes)
-    elif export_type in {
```

Comment:
This path was not tested?

Reply:
The export_type is removed in the PR so we don't need to test the other paths anymore, if that's your question?